modules/data.atmosphere/R/solar_angle.R

```diff
@@ -2,6 +2,27 @@
 #'
 #' For explanations of formulae, see https://web.archive.org/web/20180307133425/http://www.itacanet.org/the-sun-as-a-source-of-energy/part-3-calculating-solar-angles/
 #'
+#' @description
+#' `cos_solar_zenith_angle` returns the cosine of solar zenith angle.
+#' The cosine of the solar zenith angle (cossza) is the angle between the sun’s rays and the vertical
+#' This solar zenith angle is normally used in combination with the solar azimuth angle to determine the position of the Sun as observed from a given location on the surface of the Earth.
+#' The cosine of the solar zenith angle (cossza) is also a key component in Mean Radiant Temperature(MRT).
+#' It is required for photolysis.
+#'
+#' doy Day of year : It is the day when you are calculating the angle.
+#' lat Latitude : Latitude measures the distance north or south of the equator. The primary unit of latitude is degrees (°).
```

Comment:
Ok, but does the function want decimal degrees, or degrees, minutes, seconds?

Reply:
decimal degrees.